build-tools/packages/version-tools/package.json

```diff
@@ -24,7 +24,11 @@
 	"bin": {
 		"fluv": "./bin/run.js"
 	},
-	"files": ["/bin", "/lib", "/oclif.manifest.json"],
+	"files": [
```

Comment:
Mixed formatters?

Reply:
Hmm, looks like. I'll re-run the formatter.

Reply:
Yeah, need to run `pnpm format` from the build-tools root.